src/NavbarToggle.js

```diff
@@ -13,19 +16,21 @@ let NavbarToggle = React.createClass({
   contextTypes: {
     $bs_navbar_bsClass: PropTypes.string,
     $bs_navbar_onToggle: PropTypes.func,
+    $bs_navbar_expanded: PropTypes.bool
   },
 
   render() {
     let { children, ...props } = this.props;
     let {
       $bs_navbar_bsClass: bsClass = 'navbar',
-      $bs_navbar_onToggle: onToggle
+      $bs_navbar_onToggle: onToggle,
+      $bs_navbar_expanded: expanded
     } = this.context;
 
     return (
       <button type="button"
-        onClick={onToggle}
-        className={tbsUtils.prefix({ bsClass }, 'toggle')}
+        onClick={createChainedFunction(this.handleCollapsedState, onToggle)}
```

Comment:
That is true. I thought I had returned that line to the original when I removed the redundant state lines. Will remove now